package.json

```
@@ -8,22 +9,32 @@
 		"release": "run-s build update-version release:*",
 		"update-version": "dot-json distribution/manifest.json version $TRAVIS_TAG"
 	},
+	"dependencies": {
+		"@types/chrome": "0.0.88",
+		"dom-chef": "^3.6.1",
+		"element-ready": "^4.1.0",
+		"select-dom": "^5.1.0",
+		"webext-domain-permission-toggle": "^1.0.0",
+		"webext-dynamic-content-scripts": "^6.0.2"
```

This should be the `next` tag:

```suggestion
		"webext-dynamic-content-scripts": "^6.0.3-0"
```

A bugfix was sent but I haven't published the final version yet.

---

Done!